render/wgpu/src/backend.rs

```diff
@@ -345,12 +345,10 @@ impl<T: RenderTarget + 'static> RenderBackend for WgpuRenderBackend<T> {
         let enabled_features = self.descriptors.device.features();
         let available_features = self.descriptors.adapter.features() - enabled_features;
         let current_limits = &self.descriptors.limits;
-        let available_limits = &self.descriptors.limits;
```

Comment:
Why did you remove this?

Reply:
Discord discussion - nosamu was worried it was too verbose.

(I also realized it's the same info twice, when removing it. It should have been `let available_limits = self.descriptors.adapter.limits();` oops)